### PR TITLE
feat(Discord SFX): add presence

### PR DESCRIPTION
**Pull request by chimpdev**
## Description 
Added a presence files for discordsfx.com

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image

image

image

image

image
</details>

### CODE REVIEW

File: websites/D/Discord SFX/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+		"id": "957840712404193290",
+		"name": "can"
+	},
+	"service": "Discord SFX",
+	"description": {
+		"en": "Discord SFX lists some sound files uploaded by other site users that you can download and use for the soundboard of your Discord server, and also allows you to share your own."
+	},
+	"url": "discordsfx.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/T6CpbXk.png",
```
Comment (theusaf) [inline websites/D/Discord SFX/metadata.json:13]: May be difficult to see for light-mode users. Consider adding the background color.

File: websites/D/Discord SFX/presence.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
+const presence = new Presence({
+	clientId: "1119009771538882740",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			details: "Browsing",
+			largeImageKey: "https://i.imgur.com/T6CpbXk.png",
```
Comment (theusaf) [inline websites/D/Discord SFX/presence.ts:8]:
```suggestion
			largeImageKey: "https://i.imgur.com/Sr6B9gS.png",
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Review by theusaf — APPROVED** [review]

File: websites/D/Discord SFX/presence.ts [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+		}
+	}
+
+	if (pathname.startsWith("/sounds/")) {
+		presenceData.details = "Viewing Sound";
+		presenceData.state = document.title.split(" | Sound ")[1];
+
+		presenceData.buttons = [
+			{
+				label: "View on Discord SFX",
```
Comment (Bas950) [inline websites/D/Discord SFX/presence.ts:66]: View Page or View Sound

File: websites/D/Discord SFX/presence.ts [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+	if (isPlayingSound) {
+		const { name, id } = isPlayingSound.dataset;
+		let newState = "";
+
+		if (name) {
+			newState = `Listening to ${name}.mp3`;
+			if (id) {
+				presenceData.buttons = [
+					{
+						label: "View on Discord SFX",
```
Comment (Bas950) [inline websites/D/Discord SFX/presence.ts:84]: Ditto

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**